plugins/postgres/dbt/include/postgres/macros/catalog.sql

```diff
@@ -49,5 +44,28 @@
   {%- endcall -%}
 
   {{ return(load_result('catalog').table) }}
+{%- endmacro %}
 
+
+{% macro postgres__get_catalog(information_schema, schemas) -%}
```

Comment:
The `schemas` parameter to this macro is indeed a list of strings. The loop below converts it to relations: List[obj] where the obj is acting like a BaseRelation (specifically a "schema relation") for the limited purpose of calling `get_catalog_where_clause`. 

So when this macro is called with schemas = [ "schema_a", "schema_b", "schema_c" ], it will in turn call `get_catalog_where_clause` with relations = [ { "schema": "schema_a" }, { "schema": "schema_b" }, { "schema": "schema_c" } ].

This is just to make it easier to reuse get_catalog_where_clause.

Reply:
Oh, does jinja treat `relation.schema` the same as `relation.get("schema")` or `relation["schema"]`?